app/lib/analytics.coffee

```diff
@@ -0,0 +1,36 @@
+$ = require('jqueryify')
+###
+This will log a user interaction using the analytics.zooniverse.org API.
+eventData should be of the format:
+  { time: 1422379082808,  (JSON timestamp)
+  user_id: '1234',
+  subject_id: '5678',
+  type: 'logout',
+  related_id: '8901' }
+###
+logEvent = (user_id,subject_id,type,related_id) ->
```

Comment:
Since `user_id` and `subject_id` are always `User.current` and `Subject.current`, why not just assign those in the `logEvent` function? Then the log call could just be `logEvent(type, related)`.